components/alert/test/alert-toast.vdiff.js

```diff
@@ -33,4 +41,58 @@ describe('alert-toast', () => {
 			await expect(document).to.be.golden();
 		});
 	});
+
+	describe('multiple-alerts', () => {
+		async function openAlerts(elem) {
+			const alert1 = elem.querySelector('#alert-middle');
+			const alert3 = elem.querySelector('#alert-bottom');
+			alert1.open = true;
+			await alert1.updateComplete;
+			alert3.open = true;
+			await alert3.updateComplete;
+		}
+
+		it('open all', async() => {
+			const elem = await fixture(multipleAlerts, { viewport: { width: 700, height: 400 } });
+			await openAlerts(elem);
+			await expect(document).to.be.golden();
+		});
+
+		it('open all then close top', async() => {
+			const elem = await fixture(multipleAlerts, { viewport: { width: 700, height: 200 } });
+			await openAlerts(elem);
+			const alert = elem.querySelector('#alert-top');
+			alert.open = false;
+			await alert.updateComplete;
+			await expect(document).to.be.golden();
+		});
+
+		it('open all then close middle', async() => {
+			const elem = await fixture(multipleAlerts, { viewport: { width: 700, height: 300 } });
+			await openAlerts(elem);
+			const alert = elem.querySelector('#alert-middle');
+			alert.open = false;
+			await alert.updateComplete;
+			await expect(document).to.be.golden();
+		});
+
+		it('open all then close bottom', async() => {
+			const elem = await fixture(multipleAlerts, { viewport: { width: 700, height: 300 } });
+			await openAlerts(elem);
+			const alert = elem.querySelector('#alert-bottom');
+			alert.open = false;
+			await alert.updateComplete;
+			await expect(document).to.be.golden();
+		});
+
+		it('narrow', async() => {
+			const elem = await fixture(multipleAlerts, { viewport: { width: 400, height: 400 } });
+			await openAlerts(elem);
+			await expect(document).to.be.golden();
+		});
+
+		it.skip('resize', async() => {
```

Comment:
TODO?

Reply:
Yup, I'll remove until the resize work happens.